protos/remote_id/remote_id.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+
+package mavsdk.rpc.remote_id;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.remote_id";
+option java_outer_classname = "RemoteIdProto";
+
+service RemoteIdService {
+    /*
+     * Update the BasicId strucutre sent with the basic_id packet
+     */
+    rpc SetBasicId(SetBasicIdRequest) returns(SetBasicIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the Location strucutre sent with the location packet
+     */
+    rpc SetLocation(SetLocationRequest) returns(SetLocationResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the System strucutre sent with the system packet
+     */
+    rpc SetSystem(SetSystemRequest) returns(SetSystemResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the OperatorId strucutre sent with the operator_id packet
+     */
+    rpc SetOperatorId(SetOperatorIdRequest) returns(SetOperatorIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+    /*
+     * Update the SetSelfId strucutre sent with the self_id packet
+     */
+    rpc SetSelfId(SetSelfIdRequest) returns(SetSelfIdResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+message SetBasicIdRequest {
+    BasicId basic_id = 1; // Desired basic_id
+}
+message SetBasicIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetLocationRequest {
+    Location location = 1; // Desidered location
+}
+message SetLocationResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSystemRequest {
+    SystemId system = 1; // Desidered system
+}
+message SetSystemResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetOperatorIdRequest {
+    OperatorId system = 1; // Desidered operator_id
+}
+message SetOperatorIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message SetSelfIdRequest {
+    SelfId self_id = 1; // Desidered self_id
+}
+message SetSelfIdResponse {
+    RemoteIdResult remote_id_result = 1;
+}
+
+message BasicId {
+    uint32 id_type= 1;  // Indicates the format for the uas_id field of this message.
+    uint32 ua_type = 2; // Indicates the type of UA (Unmanned Aircraft).
+    string uas_id = 3;  // UAS ID following the format specified by id_type.
+}
+
+message Location {
+    uint32 status= 1;             // Indicates whether the unmanned aircraft is on the ground or in the air.
+    uint32 direction= 2;          // Direction over ground measured clockwise from true North: 0 - 35999 cdeg.
+    uint32 speed_horizontal= 3;   // Ground speed. Positive only. cm/s
```

Comment:
Our convention is that we add the unit as a suffix, e.g:

```suggestion
    uint32 speed_horizontal_cm_s= 3;   // Ground speed. Positive only. cm/s
```

Then my second question would be: can't it be `m/s`? Feels more natural to me for an aircraft :thinking: